src/backend/types/checkers.ts

```diff
@@ -0,0 +1,7 @@
+export interface CheckerData {
+    readonly status: 'enabled' | 'disabled' | 'unknown';
+    readonly name: string;
+    readonly analyzer: string;
+    readonly description: string;
+    readonly labels: string[];
+}
```

Comment:
Can you please add a new line here? Otherwise, the checker data IF looks correct.